templates/base/components/loaders/spinner.css

```
@@ -0,0 +1,138 @@
+  transform: rotate(45deg) rotateX(-385deg) rotateY(385deg);
+  border-color: #7C8FFF;
+ }
+}
+
+.loading-text {
+  display: flex;
+  align-items: center;
+  font-size: 16px;
+  color: #5B74FF;
+  letter-spacing: 0.5px;
+  font-size:18px
```

Biome (2.1.2)

[error] Duplicate properties can lead to unexpected behavior and may override previous declarations unintentionally.

font-size is already defined here.

Remove or rename the duplicate property to ensure consistent styling.

(lint/suspicious/noDuplicateProperties)